career-dev.md

```diff
@@ -0,0 +1,29 @@
+*What is your greatest strength and how do you know?
```

Comment:
Similar to the Gear Up, we'd like to see these responses expanded on. I encourage you to start telling your unique story, which you will certainly expand on and practice during your time at Turing.